lib/v8.js

```diff
@@ -80,3 +87,110 @@ exports.getHeapSpaceStatistics = function() {
 
   return heapSpaceStatistics;
 };
+
+/* V8 serialization API */
```

Comment:
Tiniest of style nits but is there a reason for mixing C and C++-style comments?